build.gradle

```diff
@@ -1,11 +1,11 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.2'
-        classpath 'me.tatarka:gradle-retrolambda:3.2.5' // use Java 8 lambdas on android
```

Comment:
Maybe need to fix the mention of retrolambda in README.md, jacoco.gradle, lint.xml and proguard-rules.pro?

Reply:
👍